packages/studio-ui/src/web/components/ContentForm/ContentNotice.tsx

```diff
@@ -0,0 +1,36 @@
+import { Callout, Intent } from '@blueprintjs/core'
+import { lang } from 'botpress/shared'
+import React, { useContext } from 'react'
+import { connect } from 'react-redux'
+import { RootReducer } from '~/reducers'
+import { FlowReducer } from '~/reducers/flows'
+import { WidgetContext } from '../Content/Select/WidgetContext'
+import { getContentItemUsage } from '../Shared/Utils'
+import withLanguage from '../Util/withLanguage'
+
+import style from './style.scss'
+
+/** Displays a warning to user if cms content used in multiple places */
+export function ContentNotice({ flows, qnaUsage }: { flows: FlowReducer; qnaUsage: any }) {
```

Comment:
The typescript-react community has mostly moved away from using the `React.FC` type, mainly because it always implies children even when components shouldn't accept them. I found a couple of sources for that in case you want to read more about it.
- https://www.harrymt.com/blog/2020/05/20/react-typescript-react-fc.html
- https://fettblog.eu/typescript-react-why-i-dont-use-react-fc/
- https://stackoverflow.com/a/61547010/903980